lib/catalog/martyrology.ts

```diff
@@ -3167,6 +3421,29 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Oliver Plunket',
       titles: [Title.Bishop, Title.Martyr],
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/saint-omer-eveque
+    // - https://en.wikipedia.org/wiki/Audomar
+    omer_audomar_bishop: {
```

Comment:
`Audomar` is not a surname, it is another variant of `Omer`. In fact, I’d somewhat prefer using `Audomar of Thérouanne` in English ([src](https://www.worldhistory.org/image/13821/audomar-of-therouanne--dagobert-i/)).

Alternatively, we could use `Omer of Thérouanne`.

Remember to update the `name` property (two lines below) and the value in `en.ts`.

```suggestion
    audomar_of_therouanne_bishop: {
```